### ISSUE TITLE
Require admins to decide whether reports are forwarded

### ISSUE THREAD

**Issue by trwnh** [mastodon/mastodon#18518]
> I think requiring the remote mods to "forward" the original report (instead of users being able to do it without oversight) would be a good compromise here. (this was how the feature was originally proposed)

While I don't agree with the specific request of #7608 I think the discussion there has generated a better solution to the root issue. I have seen some admin friends complaining about spurious or inactionable reports being sent to their instance, and I think allowing any arbitrary user to decide whether a report is delivered remotely is a mistake.

Furthermore, even if the report is "valid" at the source instance, it may not be "valid" at the remote instance, due to incompatibility between rules or simply disagreements on what constitutes a reportable offense.

It would make more sense for moderation teams to liaison with each other diplomatically rather than letting users decide that a report should be sent to a moderation team that is not responsible for them.

Semamtically this also makes more sense because reports are forwarded via the instance actor and not via the original user account. It is "more correct" for the instance admin/mod to decide whether the report is forwarded, otherwise it is like letting someone else publish activities under your own identity.

**Comment by Tealk** [comment]
But then the instance admins/mods should also be able to communicate with each other via the report tool.

Currently I often have the users are reported on other instances, then I can only close the message and hope that the other instance takes care of it.

**Comment by trwnh** [comment]
Yes, this can imply a way for admins to communicate with each other on behalf of their instances, but this is not strictly necessary because admins can just DM each other normally through the existing network.

**Comment by Tealk** [comment]
But then everything is so scattered that the information will never be found again.
I do not know how much you write with other people, but a DM is no longer findable in 4 weeks with me.

In addition, it would be important that all moderators have the same information, which is not solvable by DM.

**Comment by trwnh** [comment]
I agree for collaboration purposes this should be done between instance actors. Letting instance actors message each other more generally has other benefits as well.

**Comment by elektrikfisch** [comment]
Agree that admins should be able to decide whether their server handles, ignores, or holds for review, the 'forward' flag.

There should also be a button in the moderation interface to forward the report later, even if the reporter did not request it. If I was smart enough to implement it myself, I'd even add a mechanism for the reporting moderator to amend the information included with the forwarded report, but I'd rather have the simple forward button if adding the ability to amend the report would add a magnitude of effort because of backend work needed.

--

As the admin of a small 3-day-old instance, I'm already getting attention from so-called "free speech absolutist" instances because user reports from my instance are being forwarded to them, and I have no way to inhibit that without defederation (does defederation actually inhibit forwarding reports? I hope so), which I don't know to do until I've seen and handled the report.

The official Mastodon Android app also doesn't even give users the option whether to forward their reports. The forward flag is hard-coded. This attention from undesirable instances is how I discovered that 😆

**Comment by ThisIsMissEm** [comment]
@vmstan could you please add the moderation and activitypub labels to this? Thanks!

**Comment by nomad-geek** [comment]
imo, this is how this functionality should be implemented.

**Comment by ThisIsMissEm** [comment]
I think this is also related to: https://github.com/mastodon/mastodon/issues/24983

**Comment by rugk** [comment]
UI suggestion for an idea that leaves it in the hand of users whether to report while allowing admin to say "do only forward to these trusted servers":
> Forward reports to remote server (drop down):
> * automatically (selected by default)
> * yes forward (recommend for trusted.social)
> * no do not forward

The key point is the automatic option, how should users know whether forwarding is good or they forward to a Spammer? It also improved the UX because it makes users think less. 

Here is what the actions do:
* automatic = based on a list of trusted servers the admin of an instance can define, this automatically forwards or does not
* yes = in this case the server is on a trusted list and this this option can also be highlighted as recommend, this is the same as automatic, but the user can check their admin decision 
* no = the user, contrary to their admin view, does not want it forwarded 

If the instance were not on a trusted list, it would recommend the "no" option obviously. A user could still decide whether to allow this, though.

Inspired by [this toot](https://toot.kif.rocks/@anathem/116034880998279214)

**Comment by colinstu12** [comment]
Would there be a way to hold reports to external instances and let a mod/admin review them before they can be forwarded along? Like once report is acted upon in a specific way it could also trigger this?
